### ISSUE TITLE
`createInjector` does not get transpiled to ES5

### ISSUE THREAD

**Issue by vvanpo** [znck/vue-runtime-helpers#6]
I'm using `rollup-plugin-vue` to bundle `.vue` components, but all my generated bundles contain the following definition:

```js
function createInjector(context) {
    return (id, style) => addStyle(id, style);
}
```

And thus my bundles break in IE11.

It doesn't seem to matter if I run the `@rollup/plugin-babel` plugin after `rollup-plugin-vue`, the untranspiled output is the same.

**Comment by nlien** [comment]
I got here via this https://github.com/vuejs/rollup-plugin-vue/issues/341.
Is it possible to transpile these helpers, so we don't have to explicitly target this module for transpiling ourselves?

**Comment by nils-christian** [comment]
I agree. The arrow function is a major problem for us, as it breaks IE11 support in our application. Please transpile the helper modules.

**Comment by vvanpo** [comment]
It turns out my configuration was at fault here, I had configured Babel to exclude the entirety of `node_modules`. If I make an exception for `vue-runtime-helpers` and ensure that the Babel plugin is configured *after* `rollup-plugin-vue`, the above problem is resolved. Same is true for the `normalizeComponent` function, which I found was also not being transpiled before (although it does not contain arrow functions, just `const`, which does work in IE11).

However, I noticed that the resulting bundles are still littered with `const __vue...__ = ` definitions, indicating some part of the code generation is not being transpiled. I don't understand much about how Rollup works or how the plugin invokes Babel, but [setting the `extensions` option](https://github.com/rollup/plugins/tree/master/packages/babel#extensions) for `@rollup/plugin-babel` to include `.vue` fixes this problem.

So to summarize:
```js
// rollup.config.js

import babel from '@rollup/plugin-babel'
import vue from 'rollup-plugin-vue'

export default {
  ...
  plugins: [
    ...
    vue(), // Ensure `rollup-plugin-vue` runs before Babel.
    babel({
      exclude: new RegExp(`node_modules/(?!vue-runtime-helpers)`), // Ensure `vue-runtime-helpers` is not ignored by Babel.
      extensions: ['.js', '.jsx', '.es6', '.es', '.mjs', '.vue'],
    }),
  ],
  ...
}
```

The above configuration fixes both transpilation issues with `rollup-plugin-vue`.

@znck I'll close this issue, but I think we should document these transpilation issues and how to mitigate them.
